.github/workflows/pr-checks.yml

```
@@ -0,0 +1,107 @@
+  unit-tests:
+    name: Unit Tests
+    if: env.SHOULD_RUN == 'true'
+    uses: ./.github/workflows/_test.yml
+    with:
+      test_tags: unit
+      test_name: unit-tests
+      parallelism: 4
+
+  integration-tests:
+    name: Integration Tests
+    if: env.SHOULD_RUN == 'true'
```

actionlint (1.7.4)

context "env" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)